src/WebJobs.Extensions.DurableTask/AppServiceLinuxLoggerProvider.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    internal class AppServiceLinuxLoggerProvider : ILoggerProvider
```

Comment:
I'm not sure another ILoggerProvider is the right way to go about this, as I think the functions host already gives us an ILoggerProvider and I don't want to conflict with anything there.

We could probably put this in a new interface or just stick the logic in the extension's `Initialize()` method if we don't expect dependency injection to be required in the future. This new interface could take an `INameResolver` so it can read environment variables directly, and won't need to take a category name since we only care about environmentvariables when building our Logger.